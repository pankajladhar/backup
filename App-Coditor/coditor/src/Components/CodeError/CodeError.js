import React from "react";
import PropTypes from "prop-types";

const CodeError = ({ error }) => {
  return <pre className="error">{error}</pre>;
};

CodeError.propTypes = {
  error: PropTypes.string
};

export default CodeError;
