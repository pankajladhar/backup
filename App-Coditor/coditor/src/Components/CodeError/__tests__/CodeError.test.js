import React from "react";
import renderer from "react-test-renderer";
import CodeError from "./../CodeError";

describe("<CodeError /> ", () => {
  it("should render properly", () => {
    const tree = renderer.create(
      <CodeError error={'{"Test-1":"Fail","Test-2":"Fail","Test-3":"Fail"}'} />
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
