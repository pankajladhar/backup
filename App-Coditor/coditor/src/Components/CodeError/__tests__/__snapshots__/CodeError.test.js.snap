// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CodeError />  should render properly 1`] = `
<pre
  className="error"
>
  {"Test-1":"Fail","Test-2":"Fail","Test-3":"Fail"}
</pre>
`;
